x-pack/plugins/file_upload/public/util/geo_processing.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+
+const DEFAULT_SETTINGS = {
+  number_of_shards: 1
+};
+
+const DEFAULT_GEO_SHAPE_MAPPINGS = {
+  'coordinates': {
+    'type': 'geo_shape'
```

Comment:
reuse the constants defined in `common/constants` https://github.com/thomasneirynck/kibana/blob/e4f78950fccce08d0a64ea0c7ded4a10587c438c/x-pack/plugins/maps/common/constants.js#L34

there's a couple of places in the code where to replace this

Reply:
I added the same constants to the file_upload plugin and imported them from there